packages/nocodb/src/modules/jobs/export-import/export.service.ts

```diff
@@ -245,6 +245,9 @@ export class ExportService {
             column_name: column.column_name,
             cc: column.cc,
             cdf: column.cdf,
+            dt: column.dt,
+            dtxp: column.dtxp,
+            dtxs: column.dtxs,
```

Comment:
For now this resolves duplicate issues, later we might need to revise this in order to support operations between different db types